src/ucs/type/cpu_set.h

```diff
@@ -0,0 +1,77 @@
+/**
+* Copyright (C) Mellanox Technologies Ltd. 2001-2014.  ALL RIGHTS RESERVED.
+*
+* See file LICENSE for terms.
+*/
+
+#ifndef UCS_TYPE_CPU_SET_H
+#define UCS_TYPE_CPU_SET_H
+
+#include <stdint.h>
+#include <stddef.h>
+
+/* Type for array elements in 'ucs_cpu_set_t'.  */
+typedef unsigned long int ucs_cpu_mask_t;
+
+/* Size definition for CPU sets.  */
+#define UCS_CPU_SETSIZE  1024
+#define UCS_NCPUBITS     (8 * sizeof (ucs_cpu_mask_t))
+
+#define UCS_CPUELT(_cpu)  ((_cpu) / UCS_NCPUBITS)
```

Comment:
also we defined in code style that macro arguments should begin with _ to avoid mix with "regular" identifiers